<?xml version="1.0"?>
<launch>

  <node pkg="driver_blvd_controller" type="driver_control" name="driver_control" output="screen">
    <rosparam command="load" file="$(find agv_main)/param/robot_param/robot.yaml" />  
  </node>
  
  <!-- <node pkg="driver_blvd_controller" type="blvd20km_controller" name="Driver_motor_1" args="1 /dev/ttyS3 115200" output="screen"></node>
  <node pkg="driver_blvd_controller" type="blvd20km_controller" name="Driver_motor_2" args="2 /dev/ttyS4 115200" output="screen"></node> -->

  <node pkg="driver_blvd_controller" type="blvd20km_controller" name="driver_motor_left" args="1 /dev/ttyS3 115200" output="screen" ns="left_wheel">
    <rosparam command="load" file="$(find agv_main)/param/robot_param/driver.yaml" />  
  </node>
  <node pkg="driver_blvd_controller" type="blvd20km_controller" name="driver_motor_right" args="2 /dev/ttyS4 115200" output="screen" ns="right_wheel">
    <rosparam command="load" file="$(find agv_main)/param/robot_param/driver.yaml" />  
  </node>

</launch>